components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```diff
@@ -8389,6 +8451,27 @@ components:
         advertiseOnly:
           type: boolean
           example: true
+    APIMetadata:
+      title: API Info object with basic minimal API details.
+      type: object
+      properties:
+        id:
+          type: string
+          example: 01234567-0123-0123-0123-012345678901
+        name:
+          type: string
+          example: CalculatorAPI
+        context:
+          type: string
+          example: CalculatorAPI
+        version:
+          type: string
+          example: 1.0.0
+        provider:
+          type: string
+          description: |
+            If the provider value is not given, the user invoking the API will be used as the provider.
```

Comment:
Is this a valid description?

Reply:
Yes. It's the same description used in the APIInfo object. I just reused it

Reply:
What do you mean? `Provider` is not an input here. `APIMetadata` is basically a stripped-down version of `APIInfo`.